src/i18n/de.json

```diff
@@ -0,0 +1,72 @@
+{
+  "AUTH": {
+    "LOGIN": "Login",
+    "LOGOUT": "Logout",
+    "USERNAME": "Username",
+    "PASSWORD": "Password",
+    "REGISTER": "Register",
+    "PASSWORD_REENTRY": "Reenter Password",
+    "EMAIL": "Email",
+    "REMEMBER_ME": "Remember me",
+    "REGISTER_LINK": "No account? Register here!",
+    "PERMISSIONS": {
+      "PERMISSIONS": "Permissions",
+      "MANAGE": "Manage user permissions",
+      "LOADED": "{{username}} has the following permissions:",
+      "GET": "Get User Permissions",
+      "UPDATE": "Update User Permissions"
+    }
+  },
+  "NEWS": {
+    "NEWS": "News",
+    "POSTED_BY": "Posted by {{post.author}} on {{post.created | date:'MMM d, y'}}",
+    "LAST_EDITED": "Last edited {{post.lastEdit | date:'MMM d, y'}}",
+    "READ_MORE": "Read more",
+    "FAILED": "Failed to load posts",
+    "CREATE": "Create Post",
+    "POSTS": {
+      "IMAGE": "Image Link (Optional)",
+      "BODY": "Post Body",
+      "TITLE": "Title",
+      "TAGS": "Tags (comma-separated)",
+      "ACTIONS": {
+        "AVAILABLE": "Available Post Actions",
+        "EDIT": "Edit Post",
+        "TRANSLATE": "Translate Post",
+        "CREATE": "Create New News Post"
+      }
+    },
+    "COMMENTS": {
+      "COMMENTS": "Comments",
+      "ACTIONS": "Comment Actions",
+      "ADD": "Add comment",
+      "DELETE": {
+        "DELETE": "Delete",
+        "CONFIRMATION": "Are you sure?"
+      }
+    }
+  },
+  "PROJECTS": {
```

Comment:
At the moment, we don't have localization for the project name / description. How we should handle these translations may not be straightforward.